bindata/v4.1.0/kube-controller-manager/recycler-cm.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: openshift-kube-controller-manager
+  name: kube-controller-manager-recycler-config
+data:
+  recycler-pod.yaml:
```

Comment:
please directly embed your yaml here.  Then you can simply create this configmap.